doc/media_convert.md

```diff
@@ -1,21 +1,35 @@
 How to use the original game assets?
 ------------------------------------
 
-Openage currently depends on the original game assets, so you need a copy of the *original AoE II or AoE II: HD*. If you use *AoE II: HD*, make sure it is on patch level 4.3. You have a few options.
+Openage currently depends on the original game assets, so you need a valid copy of the *original game files*.
 
-* Use [Proton](https://github.com/ValveSoftware/Proton) which is integrated into [Steam Play](https://store.steampowered.com/linux) to download the *AoE II: HD* assets (and even play it).
+Currently we *support* conversion for these games:
+
+* **RoR**: Age of Empires 1 (1997) + Rise of Rome
+* **AoC**: Age of Empires 2 (1999) + The Conqueror's
+* **SWGB**: Star Wars: Galactic Battlegrounds + Clone Campaigns
+* **DE2**: Age of Empires 2: Definitve Edition
+
+The following versions are currently *unsupported*, but will be added later:
+
+* **HD**: Age of Empires 2 (2013) (formerly: Age of Empires 2: HD Edition)
+* **DE1**: Age of Empires 1: Definitive Edition
```

Comment:
DE 2 conversion is supported and HD isn't? I am not sure about that I had to install HD specifically to get openage to run and I think I was specifically told to do so on IRC. @heinezen

Reply:
That's okay. The new converter introduced in #1151 does not support HD yet, but it's probably gonna be implemented back in sooner or later (#1289 ).